### DISCUSSION TITLE
Bør kirby ikke understøtte en farvepalette med goldenrod og seaweedgreen

### DISCUSSION THREAD

**Discussion by bogedal** [kirbydesign/designsystem#2227] [General discussions]
Jeg tænker det vil give større kunde tilfredshed hvis kirby fik en alternativ, eller måske endda ny primær farve palette indeholdende blandt andet farven goldenrod og seaweedgreen.

Hvad tænker i?

**Comment by MadsBuchmann** [comment]
Ja. 

Alle svar der er uenige med dette post vil blive slettet, MED DET SAMME!
